src/components/CourseForm/sections/CourseFormInfo.js

```diff
@@ -64,6 +64,16 @@ export default class CourseFormInfo extends React.Component {
               name="sectionName"
             />
           </Col>
+          <Col xs={3}>
+            <Field
+              component={TextField}
+              id="edit-course-number-of-students"
+              label={<FormattedMessage id="ui-courses.field.numberOfStudents" />}
+              name="numberOfStudents"
+              parse={value => parseInt(value, 10)}
```

Comment:
Is there a trivial way to also validate that this is an integer `/^[0-9]+$/`?

Reply:
Depends on your definition of "trivial!" I added some validation, but didn't think it was obvious enough what was happening to leave it without a comment. :P